testing/vcs/test_boxfill_decreasing_latitude.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+import cdms2
+import cdutil
+import os
+import sys
+import vcs
+
+pth = os.path.join(os.path.dirname(__file__),"..")
+sys.path.append(pth)
+import checkimage
+
+f=cdms2.open(sys.argv[2])
+ice = f("variable_6")
+x=vcs.init()
```

Comment:
@danlipsa please add the bit to turn off antialiasing and increase the background size, also turn off logo. Thanks